src/query.js

```diff
@@ -147,7 +148,7 @@ function repo(req) {
     return "";
   }
 
-  const re = /^[a-zA-Z0-9\-][\w\-.]{0,213}\/[a-zA-Z0-9\-][\w\-.]{0,213}$/;
+  const re = /^[a-zA-Z\d-][\w-.]{0,213}\/[a-zA-Z\d-][\w-.]{0,213}$/;
```

Comment:
Unfortunately the changes here, result in invalid regex. `-` standalone attempts to create a range, meanwhile `\-` indicates `-` literally character, which is what we want to do. Otherwise if we move `-` to the last position of the character class, it becomes a standard character to match within the set. So either we should add back in `\-` or create a result that looks like this `^[a-zA-Z\d-][\w.-]{0,213}\/[a-zA-Z\d-][\w.-]{0,213}$`. Your choice as to which method looks best.